CONTRIBUTING.md

```diff
@@ -1,178 +1,494 @@
 # Contributing
 Contributing encompasses repository specific requirements, and the global [Quipucords contribution guidelines](https://github.com/quipucords/quipucords/blob/master/CONTRIBUTING.md).
 
-## Install
-Before developing you'll need to install:
-* [NodeJS and NPM](https://nodejs.org/)
-* [Docker](https://docs.docker.com/desktop/)
-  * Alternatively, you can try [Podman](https://github.com/containers/podman).
-* And [Yarn](https://yarnpkg.com)
+## Process
+<details>
+<summary><h3 style="display: inline-block">Using Git</h3></summary>
+
+Quipucords-UI makes use of
+- GitHub's fork and pull workflow.
+- A linear commit process and rebasing. GitHub merge commits, and squashing are discouraged in favor of smaller independent commits
```

Comment:
This information was not very clear to me as a non English speaker. Suggestion:

`A linear commit process and rebasing, as it helps maintain a clean and organized commit history. Instead of relying on GitHub merge commits and squashing, we recommend breaking down changes into smaller, independent commits.`

Reply:
I agree it is worded oddly... Not opposed to the recommendation, but I think we can simplify it a bit, have to think on it